### PR TITLE
New account setup: add support for client certificates

### PR DESCRIPTION
**Pull request by cketti**
image

### CODE REVIEW

File: feature/account/setup/src/main/kotlin/app/k9mail/feature/account/setup/ui/incoming/AccountIncomingConfigContent.kt [hunk 1]
```diff
@@ -194,22 +185,13 @@ internal fun AccountIncomingConfigContent(
                     }
 
                     item {
-                        if (state.imapAutodetectNamespaceEnabled) {
```
Comment (wmontwe) [inline feature/account/setup/src/main/kotlin/app/k9mail/feature/account/setup/ui/incoming/AccountIncomingConfigContent.kt:197]: Is this an intended change?

Comment (cketti) [reply feature/account/setup/src/main/kotlin/app/k9mail/feature/account/setup/ui/incoming/AccountIncomingConfigContent.kt:197]: No. Good catch! I think I messed up the rebase.

**Review by wmontwe — APPROVED** [review]
LGTM

In case a certificate is selected, we don't need the password anymore. Maybe we hide it or disable.

**Comment by cketti** [comment]
The app supports using a client certificate and a password at the same time. To use a client certificate instead of a password, the authentication method "Client certificate" has to be selected. We already hide the password field in that case.

**Pull request merged**